apps/docs/content/components/FAQ/index.mdx

```diff
@@ -26,7 +26,7 @@ For answers that need to provide a list of tasks, we recommend using an [ordered
 <DoDontContainer stacked>
   <Do>
     <img
-      src="https://github.com/primer/brand/assets/912236/824ebdee-4cc0-425f-8521-2c4e7b06491f"
+      src="https://github.com/primer/brand/assets/912236/c710eb6f-bbdd-43e3-85ff-18b1e0f3c259"
```

Comment:
Is it intended that the headings say 'heading' instead of 'Frequently asked questions'?

Reply:
not intended 🤦 updated!